oak/proto/enclave.proto

```diff
@@ -26,6 +26,9 @@ message InitializeInput {
   // See https://webassembly.org/docs/binary-encoding/.
   optional bytes module = 1;
   optional string node_id = 2;
+  // The port number that the enclave's GRPC server listens on; set this to 0
+  // to bind to any available port.
+  optional int32 grpc_port = 3;
```

Comment:
Could this be named `port`, or (better) rename the one on line 38 to `grpc_port` (or leave a TODO to do so)?

Reply:
SG, will rename; however I'm wondering if we should remove the field on line 38 altogether, since I assume the expectation has become that the manager specifies a port number, and EnclaveServer::Initialize should fail if that port is not available?

Reply:
Yes that's also possible, if we assume that the Oak Manager always decides which port to allocate and the Oak Node either succeeds to listen on that port or it fails entirely. @michael-kernel-sanders may also have some thoughts about this?

Reply:
I removed InitializeOutput::port in the new commit, PTAL